index.html

```diff
@@ -1049,6 +1049,14 @@ <h2>DID Parameters</h2>
 Identifies a service from the <a>DID document</a> by service ID.
 If present, the associated value MUST be an <a
 data-lt="ascii string">ASCII string</a>.
+              
+                <p class="note">
+It's generally expected that DID URL Dereferencers will interpret the service
+DID Parameter value as being mapped to a fully qualified DID URL where the <a
+  data-lt="ascii string">ASCII string</a> in the service parameter maps to a
+matching fragment identifier in the DID Document that identifies a service
```

Comment:
I think it would be preferable to improve the concrete algorithm in the DID Resolution spec (see [here](https://w3c-ccg.github.io/did-resolution/#dereferencing-algorithm-primary)), rather than adding some vague statements in DID Core..

Reply:
+1 for adding the rationale and details to the DID Resolution spec. This is really just about trying to indicate that this is common, but not that people MUST/SHOULD do things in this way. I think those types of things should fall squarely within the realm of DID Resolution.